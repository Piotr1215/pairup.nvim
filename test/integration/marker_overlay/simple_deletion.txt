Line 1
Line 2
Line 3
Line 4
Line 5
Line 6
Line 7
Line 8
Line 9
Line 10

-- CLAUDE:MARKERS:START --
CLAUDE:MARKER-2,-2 | Delete lines 2-3
CLAUDE:MARKER-6,1 | Replace line 6
Modified Line 6
CLAUDE:MARKER-8,0 | Insert after line 8
New Line After 8
CLAUDE:MARKER-10,-1 | Delete line 10
-- CLAUDE:MARKERS:END --