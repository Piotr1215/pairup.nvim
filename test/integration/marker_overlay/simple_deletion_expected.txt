Line 1
Line 4
Line 5
Modified Line 6
Line 7
Line 8
New Line After 8
Line 9